Documentation/Development/HowTo_HelpSystem.md

```diff
@@ -0,0 +1,3 @@
+# How To: Work with Help System in WheelMUD
+
+TODO.
```

Comment:
I really do think that a few things in the help system can be improved and made real time. Most of all i wan to assure that the help files can be updated, added or modified without rebooting the system, support pagination, etc.